okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -447,29 +446,48 @@ public Runnable start(Listener listener) {
           keepAliveWithoutCalls);
       keepAliveManager.onTransportStarted();
     }
+    final OkHttpClientTransport okHttpClientTransport = this;
```

Comment:
This doesn't look necessary. FYI: If you are within an anonymous class, you can do `OkHttpClientTransport.this`